kubernets/vault-auth-token.yml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: vaultauth
+type: Opaque
+data:
+  authsecret: NTE5YzNmYWUtYTZlMy1mMDhmLTViZDAtOTI3ZGJjMmY4ZDBh
```

Comment:
Yup, only exists on my laptop.  Really just left as an example of what the object needs to look like.  I'll add some info to the readme to that effect.